### PR TITLE
CPRI-4144 (Refactoring): Fetch all the ingested requests with submitted values

### PR DESCRIPTION
**Pull request by eshunsharma**
Un commenting everything.

### CODE REVIEW

File: src/services/assetRequest/mappers/ingestAssetRequests.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+export const ingestAssetRequestsMapper = (jsonResponse = {}) => {
+  const {
+    assetRequirementId,
+    isAssetRequirementOf,
```
Comment (propattern) [inline src/services/assetRequest/mappers/ingestAssetRequests.js]: planId instead of 'isAssetRequirementOf' but not a big issue.

Comment (eshunsharma) [reply src/services/assetRequest/mappers/ingestAssetRequests.js]: @propattern this is the mapper i.e. to pass data from UI to MDS, and I think MDS accepts planid as isAssetRequirementOf.

**Comment by eshunsharma** [comment]
@devu @Kerenhb @propattern All refactoring complete, it can be reviewed and merged now

File: src/services/assetRequest/createIngestAssetRequests.js [hunk 2]
```diff
@@ -14,17 +13,17 @@ import { ingestAssetRequestsDeMapper } from './mappers/ingestAssetRequests'
       submittedValues: 'Col1, Col2, Col3, Col4' // csv
     }]
 */
-/*
+
 const generatePayload = (
```
Comment (Kerenhb) [inline src/services/assetRequest/createIngestAssetRequests.js]: could be called something like deMapWithContext

File: src/services/assetRequest/mappers/ingestAssetRequests.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    submittedValues,
+    type
+  } = jsonResponse
+
+  const context = [
```
Comment (Kerenhb) [inline src/services/assetRequest/mappers/ingestAssetRequests.js]: Why do we need a context in the mapper, isn't this just going to the UI?

File: src/services/assetRequest/mappers/ingestAssetRequests.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    'https://schema.pearson.com/context/raf-context.jsonld',
+    'https://schema.pearson.com/context/context-context.jsonld'
+  ]
+
+  return {
+    context,
+    type,
+    'request-id': assetRequirementId,
```
Comment (Kerenhb) [inline src/services/assetRequest/mappers/ingestAssetRequests.js]: Could/should use @devu's constants here?

**Pull request merged**